packages/next/build/index.ts

```diff
@@ -960,8 +960,10 @@ export default async function build(
             let ssgPageRoutes: string[] | null = null
             let isMiddlewareRoute = !!page.match(MIDDLEWARE_ROUTE)
 
-            const pagePath = pagePaths.find((_path) =>
-              _path.startsWith(actualPage + '.')
+            const pagePath = pagePaths.find(
+              (p) =>
+                p.startsWith(actualPage + '.') ||
+                p.startsWith(actualPage + '/index.')
```

Comment:
What's the case of this?

Reply:
folder/index.server.js, like note/edit/index.server.js. The page is normalized so it doesn’t contain index